backup/v1.2.1/utils/downloader.py

```
@@ -0,0 +1,760 @@
+                self.finished_signal.emit(url, True, filename)
+    
+    def _add_to_downloads(self, url, success, file_path):
+        """Add download to history"""
+        now = datetime.now().strftime("%Y/%m/%d %H:%M")
+        
+        # Get file info
+        try:
+            size = os.path.getsize(file_path)
+            size_mb = size / (1024 * 1024)
+            size_str = f"{size_mb:.2f} MB"
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)